client/settings/payment-method-settings/test/digital-wallets-settings.js

```diff
@@ -4,22 +4,38 @@
  * External dependencies
  */
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import DigitalWalletsSettings from '../digital-wallets-settings';
+import {
+	useDigitalWalletsButtonActionType,
+	useDigitalWalletsButtonSize,
+	useDigitalWalletsButtonTheme,
+} from '../../../data';
```

Comment:
Honestly I can go either way with this 😅  In this specific case I prefer to use the relative imports, because `'data'` sounds like the name of an external module.
I would think about this differently if we had a Webpack alias, like `'wcpay/data'`, but in my opinion relative import makes things more explicit on identifying where the import is coming from - especially for newer people that are reading the code.
I'm not going to go and explicitly change all the imports or comment on other PRs if I see `import { thing } from 'data'`, tho 😄 
Again, if we had an alias I would feel differently :)

Reply:
In this case the deep nesting is what pushed me to suggest changing this. :)

An alias would indeed be great! Could you/should I create an issue for it?

I agree about the ambiguity - it threw me off as well when learning about this first. But I have since started checking `webpack.config.js` or whatever I need to check to make sure I get the imports right. :)

> I'm not going to go and explicitly change all the imports or comment on other PRs if I see import { thing } from 'data', tho 😄

😄 And I'd hate to do it here too! I generally stick to relative imports when components are related to each other, likely living in the same directory or in a subdirectory, or if the nesting isn't too hardcore. In this case, it is not convoluted so this is not something I'd necessarily change.